### PR TITLE
[MIG] purchase_order_line_description: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
Standard migration.
Add oca fragments for readme information.

cc @Tecnativa

### CODE REVIEW

File: purchase_order_line_description/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': "Purchase order line description",
+    'version': '11.0.1.0.0',
+    'category': 'Purchase Management',
+    'author': "Agile Business Group, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.agilebg.com',
```
Comment (nikul-serpentcs) [inline purchase_order_line_description/__manifest__.py]: @cubells URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: purchase_order_line_description/__manifest__.py (hunk 1 above)
Comment (cubells) [reply purchase_order_line_description/__manifest__.py]: @nikul-serpentcs thanks!

Done!

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please try UI when migrating, as the new config system doesn't have same structure:

seleccion_018

**Comment by cubells** [comment]
@pedrobaeza done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@nikul-serpentcs please review

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**